Portal/Controllers/WasteController.cs

```diff
@@ -93,5 +93,16 @@ public async Task<IActionResult> Types(WasteTypesViewModel wasteTypesViewModel)
 
             return RedirectToAction("Index", "Home");
         }
+
+        [HttpGet]
+        [Route("/WasteRecordStatus")]
```

Comment:
Route should proably be RecordStatus or even just Status as it's already on the WasteController

Reply:
Agreed. Actually, shouldn't we rename the controller to "Journey" or "WasteJourney"? That would be more descriptive and intuitive. Thoughts?